generators/client/templates/angular/webpack/webpack.dev.js.ejs

```diff
@@ -39,16 +39,8 @@ module.exports = (options) => webpackMerge(commonConfig({ env: ENV }), {
     devServer: {
         contentBase: './<%= DIST_DIR %>',
         proxy: [{
-            context: [<% if (authenticationType === 'uaa') { %>
-                '/<%= uaaBaseName.toLowerCase() %>',<% } %>
-                /* jhipster-needle-add-entity-to-webpack - JHipster will add entity api paths here */
-                '/api',
-                '/management',
-                '/swagger-resources',
-                '/v2/api-docs',
-                '/h2-console',<% if (authenticationType === 'oauth2') { %>
-                '/login',<% } %>
-                '/auth'
+            context: [
+                '/'
```

Comment:
Why are we removing this ? We don't need to provide those URLs with html5 mode ?

Reply:
In the past, we had to add every backend path here.  Now, every request goes to the backend and is forwarded to index.html if it doesn't match a path or resource.